lib/CL/pocl_mem_management.c

```diff
@@ -1066,34 +1066,20 @@ pocl_raw_ptr *
 pocl_find_raw_ptr_with_vm_ptr (cl_context context, const void *host_ptr)
```

Comment:
Do we still need these old functions for something if these just lock and delegate?

Reply:
There are plenty of `pocl_find_raw_ptr_{vm,dev}_ptr()` users - removing these and hoisting their code to the callers seems undesired due to code bloat it would add.